spyderlib/config/main.py

```diff
@@ -580,6 +580,51 @@ def is_ubuntu():
               # ---- Profiler (in p_profiler) ----
               'profiler/run profiler': "F10"
               }),
+            ('emacs_shortcuts',
+              {
+              # -- In codeeditor
+              'editor/go to definition': "Ctrl+G",
+              'editor/start of line': "Ctrl+A",
+              'editor/end of line': "Ctrl+E",
+              'editor/previous line': "Ctrl+P",
+              'editor/next line': "Ctrl+N",
+              'editor/previous char': "Ctrl+B",
+              'editor/next char': "Ctrl+F",
+              'editor/previous word': "Alt+B",
+              'editor/next word': "Alt+F",
+              'editor/kill to line end': "Ctrl+K",
+              'editor/kill to line start': "Ctrl+U",
+              'editor/yank': 'Meta+Y',
+              'editor/rotate kill ring': 'Shift+Ctrl+Y',
+              'editor/kill previous word': 'Ctrl+Backspace',
+              'editor/kill next word': 'Ctrl+Delete',
+              'editor/start of document': 'Alt+Left',
+              'editor/end of document': 'Alt+Right',
+              # -- In widgets/editor
+              'editor/inspect current object': 'Ctrl+X, Ctrl+I',
+              'editor/go to line': 'Alt+G+G',
+              'editor/go to previous file': 'Ctrl+Tab',
+              'editor/go to next file': 'Ctrl+Shift+Tab',
+              '_/file switcher': 'Ctrl+X, Ctrl+B',
+              # -- In spyder.py
+              'editor/find text': "Ctrl+S",
+              'editor/find next': "F3",
+              'editor/find previous': "Ctrl+R",
+              'editor/replace text': "Ctrl+X, Ctrl+H",
+              'editor/undo': 'Ctrl+_',
+              'editor/cut': 'Ctrl+W',
+              'editor/copy': 'Alt+W',
+              'editor/paste': 'Ctrl+Y',

```

Comment:
Not working either. I think the problem is these shortcuts are also defined in the Editor context menu, and so are causing conflicts with the shortcuts present in Preferences.